geojsonvt.gyp

```diff
@@ -42,6 +42,7 @@
           '<@(variant_cflags)',
           '<@(geometry_cflags)',
           '<@(rapidjson_cflags)',
+          '${CXX_FLAGS}',
```

Comment:
Oops, you're right, will fix =)

This is specifically to pass through https://github.com/mapbox/mason/blob/master/scripts/geojsonvt/4.1.2-cxx11abi/script.sh#L28